core/src/server/Config.cpp

```diff
@@ -596,6 +596,9 @@ Config::CheckCacheConfigGpuCacheCapacity(const std::string& value) {
             return s;
         }
 
+        if (gpu_index == server::CPU_DEVICE_ID)
+            return Status::OK();
```

Comment:
why need check CPU_DEVICE_ID in gpu related API ?

Reply:
This API will check if gpu_index from server_config.yaml can related to a gpu, when gpu_index is CPU_DEVICE_ID, it will return error

Reply:
rename "gpu_index" to "device_id"